modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -76,6 +77,32 @@ A clone of the Git repository is present in the filesystem of the new workspace.
 +
 The workspace has a unique URL: `pass:c,a,q[{prod-url}]#workspace__<unique_url>__`.
 
+.. Using a direct URL to `devfile`:
+[source,subs="+quotes,+attributes,+macros"]
++
+----
+pass:c,a,q[{prod-url}]#__<url_to_devfile>__
+----
++
+[TIP]
+====
+You can pass Personal Access Token to the URL to have access to private repositories:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__https://__<token>__@__<host>__/__<path> <1>
+----
+<1> See xref:using-a-git-provider-access-token.adoc[].
+
+This would work for GitHub, GitLab, Bitbucket, Microsoft Azure, and other providers that support Personal Access Token.
+====
++
+.A URL for starting a new workspace from devfile location
```

Comment:
This section gets rendered as an example. Is it the correct formatting? 
![image](https://user-images.githubusercontent.com/80100533/221644435-8ca8b2fb-30f6-400c-a7fe-5cc1514a3f89.png)

Reply:
Yes, this is an example with a Github link

Reply:
understood, thanks!